### ISSUE TITLE
gen_gettext_stats.py does not work when compiling Naev in MSYS2 on Windows.

### ISSUE THREAD

**Issue by ProjectSynchro** [naev/naev#2162]
For some reason, MSYS2's distribution of `xgettext` does not appear to be happy when this script calls on it, blocking compilation on Windows.

Compile snippet from Discord:
```
[250/256] Generating gettext_stats with a custom command
FAILED: naev.txt
"C:/msys64/mingw64/bin/python.exe" "C:/msys64/home/throstur/naev/utils/build/gen_gettext_stats.py" "--output" "naev.txt" "../po/naev.pot"
Traceback (most recent call last):
  File "C:\msys64\home\throstur\naev\utils\build\gen_gettext_stats.py", line 23, in <module>
    main()
  File "C:\msys64\home\throstur\naev\utils\build\gen_gettext_stats.py", line 16, in main
    out = subprocess.check_output(cmd, env=env, encoding='ascii', stderr=subprocess.STDOUT)
  File "C:/msys64/mingw64/lib/python3.9/subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "C:/msys64/mingw64/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['msgfmt', '--statistics', '../po/naev.pot', '-o', '/dev/null']' returned non-zero exit status 1.
[251/256] Compiling C object naev.exe.p/src_nlua_transform.c.obj
[252/256] Compiling C object naev.exe.p/src_nlua_tk.c.obj
[253/256] Compiling C object naev.exe.p/src_nlua_var.c.obj
[254/256] Compiling C object naev.exe.p/src_nlua_vec2.c.obj
ninja: build stopped: subcommand failed.
```

**Comment by UncombedCoconut** [comment]
Oh crap, it's probably the "/dev/null" output destination, isn't it? The fix might be as simple as replacing `'/dev/null'` with `os.devnull` -- but this would have to be tested!

**Comment by ProjectSynchro** [comment]
Well that was quick 😅 
That appears to have done the trick. Fixed as of 099b24572d4cc05ebbe24f284af3b7f755e1aca9.
